### ISSUE TITLE
Bug(core): Select cannot be opened with keyboard

### ISSUE THREAD

**Issue by mikerodonnell89** [SAP/fundamental-ngx#6160]
Due to https://github.com/SAP/fundamental-ngx/pull/5854/ the select popover options cannot be opened with keyboard.
